modules/virt-enabling-disabling-downward-metrics-feature-gate-yaml.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * virt/monitoring/virt-exposing-downward-metrics.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enabling-disabling-downward-metrics-feature-gate-yaml_{context}"]
+= Enabling or disabling the downward metrics feature gate in a YAML file
+
+To expose downward metrics for a host virtual machine, you can enable the `downwardMetrics` feature gate by editing a YAML file. Enabling the feature gate requires administrator privileges.
+
+.Procedure
+
+You can enable or disable the `downwardMetrics` feature gate as follows:
```

Comment:
I don't think we typically introduce procedures with a sentence like this once the `.Procedure` section has started. I get that you're trying to be consistent with the other procedure, but I think this sentence's meaning is already implied and it can probably be deleted:
```suggestion
```